civihr_default_theme/scss/base/_variables.scss

```diff
@@ -112,3 +112,8 @@ $crm-scrollbar-thumb-color: $crm-gray-light-blue !default;
 $crm-customselect-height:        33 !default;
 $crm-customselect-caret-width:   32 !default;
 $crm-customselect-padding-right: $crm-customselect-caret-width + 12 !default;
+
+// Fonts
+// -----------------------------------------------------------------------------
+$font-weight-medium: 500;
```

Comment:
Also there is already a Typography section, you should put these variables there